server.js

```diff
@@ -1,132 +0,0 @@
-import cookieParser from 'cookie-parser';
```

Comment:
You'll recognize this as a simple Express file. Everything has been replaced except for the CSRF module. I'll make that into a different PR.